make/mt-os/mt-stm32fx.mk

```diff
@@ -44,7 +44,8 @@ XI_COMPILER_FLAGS += -ffunction-sections
 # loop optimization
 XI_COMPILER_FLAGS += -fno-move-loop-invariants
 # define the board type
-XI_COMPILER_FLAGS += -DSTM32F407xx
+# XI_COMPILER_FLAGS += -DSTM32F407xx
+XI_COMPILER_FLAGS += -DSTM32F429xx
```

Comment:
So this is the guy we have to change every time we need different board.... What to do with that guy ? He is not very handy.... Maybe we could pass this as a makefile variable ? Like XI_BOARD_SYMBOL ?= STM32F407xx and then we could overwrite it ? Dunnno. But it seems that we have to be able to handle this somehow. Another idea is to use separate board-focused.mk files and include them from here using name of the board or smth. I know it rises the complexity but we need a way to handle different board versions.